meetings/2021-06-23.md

```diff
@@ -0,0 +1,125 @@
+# 2021-06-23 Solid Authorization
+
+https://meet.jit.si/solid-authorization
+
+
+## Agenda
+
+* Minutes https://github.com/solid/authorization-panel/pull/227
+* [WAC Editor's Draft PR](https://github.com/solid/web-access-control-spec/pull/83) - we can do a 5-10m (max) Q&A if of interest - csarven
+* [Consent workflow use case](https://github.com/solid/authorization-panel/pull/231)
+* [UMA like use-case discussion](https://github.com/solid/authorization-panel/discussions/230)
+
+
+## Present
+
+* Henry Story
+* Matthieu Bosquet
+* Justin Bingham
+* Aaron Coburn
+* Sarven Capadisli
+* Elf Pavlik
+* Eric Prud'hommeaux
+
+
+## Minutes
+
+### WAC PR
+
+Henry: Are the issues linked [there]( https://github.com/solid/web-access-control-spec/pull/83) gonna get closed?
+
+Sarven: Some. Everything that makes an authorization effective for example. Is an ACL mode required... Issues about extensions don't close. Some issues linked need to be addressed such as Client ID. If spec has an answer for it, we'll close.
+
+Henry: I'm interested in extending WAC in simple ways such as adding imports, how do I go around doing that?
+Should we just clone and edit/improve on it or add little specs for little pieces and add links as extensions?
+Some extensions might make things easier.
+
+Sarven: It will not be in this editor's draft that we would put for example imports. The best would be to create an issue specifying the use case in the WAC spe repo.
```

Comment:
```suggestion
Sarven: It will not be in this editor's draft that we would put, for example, imports. The best would be to create an issue specifying the use case in the WAC spec repo.
```